### DISCUSSION TITLE
How to Remove Irrelevant Entity detected after Prediction from NER model?

### DISCUSSION THREAD

**Discussion by karndeepsingh** [explosion/spaCy#9220] [Help: Coding & Implementations] [answered]
Hi,
I have trained the NER + Relation model together. I am able to get the required entities but there are extra unwanted Entity detection happening when I do the prediction on text. Following Image show's that QUOTES ( " ) are detected as IDCODE but While annotating the data I haven't annotated Quotes as IDCODE. Please help me to get rid of these kind of extra unecessary predictions. Thanks.
![image](https://user-images.githubusercontent.com/49562460/133480056-6d1ad326-1e1e-47b8-a271-2c4c1647a640.png)

![image](https://user-images.githubusercontent.com/49562460/133480899-433d376e-f47f-4fa4-bd24-515578dcb11b.png)

@polm  
@svlandeg

**Comment by svlandeg** [comment]
Hi, please refrain from pinging specific maintainers, as it makes our GH mentions practically unusable. Thanks.

**Comment by svlandeg** [comment, marked as answer]
Hi,

This answer refers to https://github.com/explosion/spaCy/discussions/9229 too.

I'm afraid we can't really help out with these kind of generic questions on Machine Learning performance. It all depends on your dataset size, annotation guidelines, variability in the data/predictions, etc etc.

In the spaCy docs, you can find information on how to tune/retrain machine learning pipelines, as well as on how to combine them with rule-based components to obtain the right balance of precision/recall your application requires. 

I'd like to add that opening multiple threads on similar issues, and pinging maintainers in various threads and discussions, doesn't help. We do our best to help anyone on these forums, but please be advised that this is on a best-effort basis. It feels like perhaps you need more help than we have the bandwidth for, so you might consider talking to a ML/NLP consultant instead.
